### ISSUE TITLE
PHP 7 issue

### ISSUE THREAD

**Issue by socketman2016** [allegro/php-protobuf#77]
I clone php 7 branch (https://github.com/serggp/php-protobuf/tree/php7) and make & make install it , protobuf.so file created successfully 
When I load extension and run php this error shown

> PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib64/php/modules/protobuf.so' - /usr/lib64/php/modules/protobuf.so: undefined symbol: Z_TYPE_PP in Unknown on line 0

```
PHP 7.0.8 (cli) (built: Jun 23 2016 17:01:31) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.8, Copyright (c) 1999-2016, by Zend Technologies

Centos 6
```

**Comment by serggp** [comment]
@socketman2016 Thank you for your report. The issue has been fixed. Please pull the branch again and try to use it.

**Comment by amereii** [comment]
Thanks a lot

**Comment by socketman2016** [comment]
Thanks a lot

**Comment by socketman2016** [comment]
Thanks a lot 

fixed #77
